app/command_utils.cc

```diff
@@ -11,7 +11,7 @@
 #include "chrome/app/chrome_command_ids.h"
 #include "brave/app/brave_command_ids.h"
 
-namespace shortcuts {
+namespace commands {
 std::vector<uint16_t> GetCommands() {
```

Comment:
while we're here, you can make it `constexpr` and don't create `std::vector` on each call.
```
base::span<const uint16_t> GetCommands() {
  static constexpr uint16_t kCommands[] = { ... };
  return kCommands;
}
```

Reply:
I'm using a map now, but I am still creating the vector on each call. Do you have a suggestion on how to avoid it? FWIW, this isn't going to be invoked too often (at the moment, only from the Command WebUI page).

Reply:
I don't think we can do something with this map now. Just make sure:
1. GetComponentInfo() is in anonymous namespace.
2. std::map is base::NoDestructor<std::map>